beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -2937,25 +2947,6 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             );
         }
 
-        // Do not store metrics if the block was > 4 slots old, this helps prevent noise during
-        // sync.
-        if block_delay_total < self.slot_clock.slot_duration() * 4 {
-            // Observe the delay between when we observed the block and when we imported it.
-            let block_delays = self.block_times_cache.read().get_block_delays(
-                block_root,
-                self.slot_clock
-                    .start_of(current_slot)
-                    .unwrap_or_else(|| Duration::from_secs(0)),
-            );
-
-            metrics::observe_duration(
-                &metrics::BEACON_BLOCK_IMPORTED_OBSERVED_DELAY_TIME,
-                block_delays
-                    .imported
-                    .unwrap_or_else(|| Duration::from_secs(0)),
-            );
```

Comment:
I moved this section of code in with the other metrics because I was getting strange outliers for import delay that I thought might have been because of the difference in observation point. However I think this code is actually fine as is, and I must have been because of blocks off the canonical chain.

I'll do some more measurements before we merge.